### PR TITLE
Update weighted-code-coverage in the CI

### PR DESCRIPTION
**Pull request by alexle0nte**
Changes:
- Update of `weighted-code-coverage` CI in templates and project workflow.
- Addition of badge and update of README template: #3.

### CODE REVIEW

File: templates/cargo/README.md [hunk 1]
```diff
@@ -2,18 +2,23 @@
 
 [![Actions Status][actions badge]][actions]
 [![CodeCov][codecov badge]][codecov]
+[![Wcc][wcc badge]][wcc]
 [![LICENSE][license badge]][license]
 
 {{ name }}'s description
 
+**Note that**: the created Github Actions workflow uses a static code analysis tool, called `weighted-code-coverage`, which produces an `html` report that is hosted on [Github Pages](https://docs.github.com/en/pages). However, to make this work you need to set from the repository settings, following these [steps](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow), Github Actions as source for Github Pages build and deployment.
+
```
Comment (Luni-4) [inline templates/cargo/README.md:11]: Change:

However, to make it work, you need to set the select box to `GitHub Actions` in the `GitHub Pages` panel contained in `Settings`, as described [here](link to steps...)

File: templates/cargo/github/test.yml [hunk 2]
```diff
@@ -330,23 +343,143 @@ jobs:
...
+      id: metrics
+      env:
+        JSON: {{ '${{ steps.wcc-run.outputs.json }}' }}
+      run: |
+        content=`cat ${JSON}`
+        echo "json=$content" >> $GITHUB_OUTPUT
+
+    - name: Extract branch name
+      id: extract-branch
+      if:  github.event_name != {{ '\'pull_request\'' }}
```
Comment (Luni-4) [inline templates/cargo/github/test.yml]: Is this necessary?  Can we just write `if:  github.event_name != 'pull_request'`? Or use another way to tell that we are in a PR?

**Review by Luni-4 — CHANGES_REQUESTED** [review]
Just some small changes

File: templates/cargo/github/test.yml (hunk 2 above)
Comment (alexle0nte) [reply templates/cargo/github/test.yml]: It works with `if: github.event_name != 'pull_request'`.

**Pull request merged**